.github/workflows/e2e-azure.yaml

```diff
@@ -0,0 +1,64 @@
+name: e2e-azure
+
+on:
+  workflow_dispatch:
```

Comment:
Can you please add here branch `azure/e2e` to see if the tests are passing?

Reply:
Tests have been run fully without any issues.